CoffeeChat/Cells/SimpleOnboardingTableViewCell.swift

```diff
@@ -50,14 +58,29 @@ class GoalTableViewCell: UITableViewCell {
             make.top.bottom.equalToSuperview().inset(5)
         }
 
-        titleLabel.snp.remakeConstraints { make in
+    }
+
+    func configure(with title: String, type: SimpleOnboardingCellType, subtitle: String?) {
+        titleLabel.text = title
+        isInterest = type == .interest
+        subtitleLabel.isHidden = type != .interest
+        if let subtitle = subtitle {
+            subtitleLabel.text = subtitle
+        }
+
+        titleLabel.snp.makeConstraints { make in
             make.leading.trailing.equalToSuperview().inset(8)
-            make.centerY.equalToSuperview()
+            if isInterest {
```

Comment:
could we potentially condense the `type` and `subtitle` arguments to just the subtitle? looking at it, it seems like `type = .interest` and having a subtitle always coincide with each other, but I don't know all the scenarios where this cell is used
this would make the cell more generic in case pear ever wants to use this cell with a subtitle, but the cell isn't `type = .interest`